cba_settings.sqf

```diff
@@ -199,8 +194,6 @@ force ace_ui_allowSelectiveUI = true;
 force ace_vehiclelock_defaultLockpickStrength = 10;
 force ace_vehiclelock_lockVehicleInventory = true;
 force ace_vehiclelock_vehicleStartingLockState = -1;
-force ace_viewdistance_enabled = true;
-force ace_viewdistance_limitViewDistance = 10000;
```

Comment:
wait, gabs nich irgendwelche fuckiness, daß aufm server die höchstmögliche view distance gesetzt sein muß, damit die clients nicht begrenzt werden?

Reply:
@gruppe-adler/adler brauch hier mal gesammelte Erfahrungswerte

Reply:
ftr: der Gedanke damals war, dem DAU der zum ersten mal ACE in die Hand nimmt, eine vernünftige Einstellung zu geben.
 
Da aber hier: https://github.com/acemod/ACE3/blob/be09fd10a3e51cdd90bf8e3d2df6f000fdec2f5a/addons/viewdistance/ACE_Settings.hpp#L5 das per default an ist, können wir das setting tatsächlich rausschmeißen 

👍

Reply:
Perfekt :)